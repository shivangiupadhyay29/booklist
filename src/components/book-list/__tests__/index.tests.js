import React from 'react';
import BookList from '../index.js';
import { shallow } from 'enzyme';

describe('BookList component', () => {
  it('BookList component unit testing', () => {
    const mockBookObject = {
      id: 0,
      title: 'book title',
      author: 'author name',
      summary: 'summary of the book text'
    };
    const mockList = [mockBookObject];
    const wrapper = shallow(<BookList books={mockList} />);
    expect(wrapper).toMatchSnapshot();
  });
});
