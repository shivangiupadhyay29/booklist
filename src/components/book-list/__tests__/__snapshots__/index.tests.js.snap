// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookList component BookList component unit testing 1`] = `ShallowWrapper {}`;
